src/app/workflow-app/workflow-app.component.ts

```diff
@@ -132,6 +157,24 @@ export class WorkflowAppComponent implements AfterContentInit {
     this.selectedWorkflow = workflow;
   }
 
+  onPopupMenuSelected(event): void {
```

Comment:
Never write logic that depends on user-visible strings. This will fail when run in a different language